src/state.ts

```diff
@@ -418,21 +418,26 @@ export class FormState<
   }
 
   @action
-  async save(options: ValidateOptions = {}): Promise<boolean> {
+  async save(
+    options: ValidateOptions = {},
+    skipValidations = false
+  ): Promise<boolean> {
     if (this.processor == null) {
       throw new Error("Cannot save without backend configuration");
     }
-    let extraOptions = {};
-    if (this.processor.process == null) {
-      extraOptions = { ignoreGetError: true };
-    }
-    const isValid = this.validate({ ...extraOptions, ...options });
+    if (!skipValidations) {
+      let extraOptions = {};
+      if (this.processor.process == null) {
+        extraOptions = { ignoreGetError: true };
+      }
+      const isValid = this.validate({ ...extraOptions, ...options });
 
-    if (!options.ignoreSaveStatus) {
-      this.setSaveStatus("rightAfter");
-    }
-    if (!isValid) {
-      return false;
+      if (!options.ignoreSaveStatus) {
```

Comment:
Should this be inside the skipValidations if-block or outside of it?

Reply:
I think yes, as the "rightAfter" means "right after validation" - we do not use validation so we shouldn't go to that state right?